crates/network-scanner/Cargo.toml

```diff
@@ -8,12 +8,20 @@ publish = false
 
 [dependencies]
 anyhow = "1.0.79"
+crossbeam = { version = "0.8.4", features = ["crossbeam-channel"] }
+dashmap = "5.5.3"
+derive_builder = "0.13.0"
+futures = { version = "0.3.30" }
+lazy_static = "1.4.0"
+network-interface = "1.1.1"
 network-scanner-net ={ path = "../network-scanner-net" }
 network-scanner-proto ={ path = "../network-scanner-proto" }
+serde = { version = "1.0.196", features = ["serde_derive"] }
```

Comment:
question: Why do we need serde in this crate?

Reply:
it was easier for the main devolutions bin to use as query parameter, removed for now